### PR TITLE
Component/Navigation

### LINKED ISSUE #4: Navigation Component
Bildschirmfoto 2021-11-15 um 11 47 57

### PR DESCRIPTION
**Pull request by Alkawi**
closes #4 

We built the navigation bar. It shows which page is active.

Bildschirmfoto 2021-11-16 um 14 48 41

### CODE REVIEW

File: src/app/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import BookmarkIcon from '../Icons/BookmarkIcon';
+import SearchIcon from '../Icons/SearchIcon';
+import HomeIcon from '../Icons/HomeIcon';
+
+export default function Navigation(): JSX.Element {
```
Comment (phlgr) [inline src/app/components/Navigation/Navigation.tsx:8]:
```suggestion
export default function Navigation({activeLink}: NavigationProps): JSX.Element {
```

File: src/app/components/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import BookmarkIcon from '../Icons/BookmarkIcon';
+import SearchIcon from '../Icons/SearchIcon';
+import HomeIcon from '../Icons/HomeIcon';
+
```
Comment (phlgr) [inline src/app/components/Navigation/Navigation.tsx:7]:
```suggestion
type NavigationProps = {
  activeLink: "home" | "search" | "bookmark";
```

File: src/app/components/Navigation/Navigation.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      <li>
+        <Link
+          to="/"
+          onClick={() => {
+            setIsHomeActive(true);
+            setIsBookmarksActive(false);
+            setIsSearchActive(false);
+          }}
+        >
+          <HomeIcon stroke={isHomeActive ? 'yellow' : 'white'} />
```
Comment (phlgr) [inline src/app/components/Navigation/Navigation.tsx:23]: Not only the stroke should be yellow, but also the fill property

File: src/app/components/Navigation/Navigation.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import BookmarkIcon from '../Icons/BookmarkIcon';
+import SearchIcon from '../Icons/SearchIcon';
+import HomeIcon from '../Icons/HomeIcon';
+
+export default function Navigation(): JSX.Element {
+  const [isHomeActive, setIsHomeActive] = useState<boolean>(true);
+  const [isSearchActive, setIsSearchActive] = useState<boolean>(false);
+  const [isBookmarksActive, setIsBookmarksActive] = useState<boolean>(false);
```
Comment (phlgr) [inline src/app/components/Navigation/Navigation.tsx:11]: These should not be in here, as the component is supposed to be a controlled component

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by marwinburesch — APPROVED** [review]
We'll talk about the "yellow" in a private session

**Pull request merged**